custom/opm/reports.py

```diff
@@ -1112,7 +1116,7 @@ def rows():
                 for method, __, __, denom in self.model.method_map:
                     value = getattr(awc, method)
                     row.append(value)
-                    if denom != 'no_denom':
+                    if denom != 'no_denom' and denom != 'one':
```

Comment:
Sounds reasonable.  What would the percentages represent?  These are all for service availabilities, right?  So that'd be:
>row: Was there a VHND held?  yes/no, percentage not sensible
total: How many AWCs held a VHND?  numerical, percent makes sense.

Is that correct?